react.gradle

```diff
@@ -76,7 +76,9 @@ gradle.projectsEvaluated {
                 workingDir reactRoot
 
                 // Set up dev mode
-                def devEnabled = !targetName.toLowerCase().contains("release")
+                def devEnabled = !(config."devDisabledIn${targetName}"
+                    || config."devDisabledIn${buildTypeName.capitalize()}"
+                    || targetName.toLowerCase().contains("release"))
```

Comment:
This is just the same as:
```
def devEnabled = !(config."devDisabledIn${targetName}"
        || targetName.toLowerCase().contains("release"))
```

productFlavorName is an empty string if no flavors are specified, making `targetName` the same as `buildTypeName`
Could you remove line 80?